packages/openid4vci/src/credential-request/format-payload.ts

```diff
@@ -37,7 +38,10 @@ export function getCredentialRequestFormatPayloadForCredentialConfigurationId(
     options.credentialConfigurationId
   )
 
-  if (zIs(zSdJwtVcCredentialIssuerMetadataDraft14, credentialConfiguration)) {
+  if (
+    zIs(zSdJwtVcCredentialIssuerMetadataDraft16, credentialConfiguration) ||
```

Comment:
For draft 16 i don't think it's possible anymore to request by format and vct right? I think only using credential_configuration_id, so this will never make sense? Or is it because we transform to draft 16 even if the issuer uses draft 14 and we should send the request as draft 14

Reply:
It's because we transform

Reply:
Agreed!